### ISSUE TITLE
Wifi add/connect screens blank

### ISSUE THREAD

**Issue by markwheeler** [monome/norns#743]
Testing the 2.0 beta I've seen these situations happen:

- Mostly: Turn on norns, wifi does connect automatically OK. Selecting 'add' shows a blank screen (waited up to 60secs, nothing appears). Selecting 'connect' correctly lists previously used network.

- Occasionally: Turn on norns, wifi does not auto connect, status reads 'Unavailable'. Select 'connect' or 'add' from the wifi menu, both are blank screens.

- The first time I added a network it worked correctly, the 'add' screen populated immediately. I've not been able to get the 'add' screen to populate again.

Seems like a few people on lines have mentioned similar. Let me know how I can help debug further.

**Comment by okyeron** [comment]
would it be helpful to do some error checking on  `m.wifi.ssid_list` and `wifi.conn_list` and display something like "can't find networks" instead of a blank screen?

**Comment by tehn** [comment]
@okyeron yes that is a good idea

**Comment by okyeron** [comment]
Looking at this again today and it seems like maybe a good solution is to add the error checking in `listselect.lua` `ls.enter()` so if `ls.len ==0` then it displays `none` or `empty` or something?

**Comment by okyeron** [comment]
Testing this just now - I got to a state where CONNECT/ADD/DEL all showed none so I rebooted. Then I would get `none` for CONNECT, but got a listing for ADD. Adding my usual network ended up with `Network 1` and then the CONNECT list showed `Network` and `Network 1` (along with another saved network)

So... it seems like the connect listing is forgetting known networks when the `wifi.status` is unavailable?
